ospo-mindmap/Content/ospomindmap.md

```diff
@@ -293,6 +293,29 @@ incubation and post launch
 - OW2
   - Good Governance Initiative
 
+### 🤝 Collaborate with Internal Stakeholders
+
+  - Legal
+    - License compliance
+    - Policies and guidelines
+    - Mergers and acquisitions
+    - Investment due diligence
+    - Contribution reviews
+    - Trademark searches for new open source projects
+  - Security
```

Comment:
Selection criteria for open source components.

Reply:
Added